### PR TITLE
Scala3doc: Add icons to symbols on page

### PR DESCRIPTION
**Pull request by pikinier20**

### CODE REVIEW

File: scala3doc/resources/dotty_res/styles/scalastyle.css [hunk 1]
```diff
@@ -613,6 +613,55 @@ footer .pull-right {
   position: relative;
 }
 
+.header {
```
Comment (romanowski) [inline scala3doc/resources/dotty_res/styles/scalastyle.css:616]: This adds a padding to lists of members as well

**Review by romanowski — CHANGES_REQUESTED** [review]

**Comment by romanowski** [comment]
Can we also include icons in navigation? If so can we make a `css` classes more compact? Navigation is added to all pages so it should be as small as possible in terms of size

File: scaladoc/src/dotty/tools/scaladoc/renderers/HtmlRenderer.scala [hunk 2]
```diff
@@ -158,7 +167,7 @@ class HtmlRenderer(rootPackage: Member, val members: Map[DRI, Member])(using ctx
           val attr = if expanded || isSelected then Seq(cls := "expanded") else Nil
           (isSelected || expanded) -> div(attr)(
             linkHtml(expanded),
-            span(),
+            span(cls := "ar"),
```
Comment (BarkingBad) [inline scaladoc/src/dotty/tools/scaladoc/renderers/HtmlRenderer.scala:170]: What `ar` name comes from? Maybe we would like to name it more meangingful?

**Review by BarkingBad — APPROVED** [review]

File: scaladoc/src/dotty/tools/scaladoc/renderers/HtmlRenderer.scala (hunk 2 above)
Comment (romanowski) [reply scaladoc/src/dotty/tools/scaladoc/renderers/HtmlRenderer.scala:170]: arrow. We will be adding tons of this to all pages so we want to keep it as short as possible

**Review by romanowski — APPROVED** [review]

**Pull request merged**